_posts/2018-11-17-elections-2019.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Announcing the third Vox Pupuli Elections
+date: 2018-11-17
+github_username: binford2k
+twitter_username: binford2k
+---
+
+Vox Pupuli has opened elections for the 2019 calendar year Project Management Committee.
+The elections were announced at Puppet's [Contributor Summit Online][cso].
+This blogpost will give you the context and how to become a candidate.
+
+> Vox Pupuli is a collective of Puppet module, tooling and documentation authors
+> all working together to ensure continued development on the code we maintain.
+
+(Source: [https://voxpupuli.org/](https://voxpupuli.org/))
+
+The Vox Pupuli project is now more than three years old. It represents many
+different collaborators, and numerous repositories of tools, plugins, and
+modules. In order to make this project successful in the long term, we elect a
+list of PMC officers to make big decisions and guide the organization in the
+right direction. This is all laid out in the [governance document][g].
+
+TL;DR The Committees members are elected by the community for 1 year. They take
+care of the Code of Conduct and its values, participate in strategic planning,
+amongst other things. [Read more about the committee and its selection][elections].
+
+The PMC has set [Kris Bosland][kris] to be the Election Officer and to run the
+elections, as stated in the [Governance Document][gd]. Kris is not running
+for PMC re-election.
+
+Nominate yourself or someone else by following the directions [here][e].
+
+Important Dates:
+
+* Before *November 30 2018 23:59UTC*: Accepting nominations
+* Between *December 1 2019 12:00UTC* and *December 15 2018 23:59UTC*: Vote
```

Comment:
depending on the amount of votes we get, I suggest extending this.  But this is probably something Kris can decide during December.

Reply:
Yeah, I figured that two week periods (with another two weeks before terms take effect) would give us a bit of a buffer.